### PR TITLE
ruby: hermitise rubygems

### PR DESCRIPTION
**Pull request by quad**

### CODE REVIEW

**Review by alecthomas — APPROVED** [review]

File: ruby.hcl [hunk 1]
```diff
@@ -3,6 +3,11 @@ strip = 2
 binaries = ["bin/*"]
 test = "ruby --version"
 
+env = {
+  GEM_HOME: "${HERMIT_ENV}/.hermit/ruby/gem",
+  PATH: "${GEM_HOME}/bin:${PATH}",
```
Comment (quad) [inline ruby.hcl:8]: @aat may I refer to the above defined `${GEM_HOME}` here? It works locally but maybe that's undefined behaviour?

Comment (alecthomas) [reply ruby.hcl]: Ah I see what you were asking about. This will actually work, because Hermit iteratively resolves envars.

Comment (quad) [reply ruby.hcl]: Iteratively as in it recognises the dependencies between them?

Comment (quad) [reply ruby.hcl]: Will https://github.com/cashapp/hermit-packages/pull/185 then?

**Pull request merged**